specification/dartLangSpec.tex

```diff
@@ -13536,23 +13539,26 @@ \section{Libraries and Scripts}
 
 \begin{grammar}
 <topLevelDefinition> ::= <classDefinition>
+  \alt <mixinDeclaration>
   \alt <enumType>
   \alt <typeAlias>
   \alt \EXTERNAL{}? <functionSignature> `;'
   \alt \EXTERNAL{}? <getterSignature> `;'
   \alt \EXTERNAL{}? <setterSignature> `;'
   \alt <functionSignature> <functionBody>
-  \alt <type>? \GET{} <identifier> <functionBody>
-  \alt <type>? \SET{} <identifier> <formalParameterList> <functionBody>
-  \alt (\FINAL{} | \CONST{}) <type> <staticFinalDeclarationList> `;'
-  \alt <variableDeclaration> `;'
+  \alt <getterSignature> <functionBody>
+  \alt <setterSignature> <functionBody>
+  \alt (\FINAL{} | \CONST{}) <type>? <staticFinalDeclarationList> `;'
+  \alt \LATE{} \FINAL{} <type>? <initializedIdentifierList> `;'
+  \alt <topLevelVariableDeclaration> `;'
 
-<getOrSet> ::= \GET{}
-  \alt \SET{}
+<topLevelVariableDeclaration> ::=
```

Comment:
Any reason this is a separate definition instead of inline like the rest?

Reply:
No, inlined it.